doc/manuals/user/walkthrough_apiv1.md

```diff
@@ -319,11 +319,11 @@ The updateContext request payload contains a list of contextElement
 elements. Each contextElement is associated to an entity (whose
 identification is provided in the entityId element, in this case we
 provide the identification for Room1) and contains a list of
-contextAttribute elements ('attributes' for short, in JSON). Each
-contextAttribute provides the value for a given attribute (identified by
-name and type) of the entity. Apart from the list of contextElement
-elements, the payload includes also an updateAction element. We use
-APPEND, which means that we wa      nt to add new information.
+attributes elements . Each contextAttribute provides the value 
+for a given attribute (identified by name and type) of the entity. 
```

Comment:
"(identified by name and type)" -> "(identified by name)"

Ancient versions of Orion used name and type to identify the attribute, but this was changed (I'm not sure, but maybe it was in 0.17.0) and now only name is used.